src/components/ArtifactDetailPage/ArtifactDetailPage.tsx

```diff
@@ -0,0 +1,139 @@
+
+  return (
+    <Grid hasGutter>
+      <GridItem xl={12} xl2={6}>
+        <Toolbar>
+          <ToolbarItem>
+            <TextContent>
+              <Text component={TextVariants.h2}>Details</Text>
+            </TextContent>
+          </ToolbarItem>
+        </Toolbar>
+        <ContentBox borderTop padding contentBoxHeight={`calc(100% - 78px)`}>
+          <Attributes>
+            <AttributesItem title={artifactEntityAttributes.filename.title}>
+              {serviceContainerArtifact.data?.filename}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.purl.title}>{serviceContainerArtifact.data?.purl}</AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.size.title}>
+              {serviceContainerArtifact.data?.size && calculateFileSize(serviceContainerArtifact.data.size)}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.artifactQuality.title}>
+              <ArtifactQualityLabelMapper quality={serviceContainerArtifact.data?.artifactQuality} />
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.importDate.title}>
+              {serviceContainerArtifact.data?.importDate &&
+                createDateTime({ date: serviceContainerArtifact.data.importDate }).custom}
+            </AttributesItem>
+          </Attributes>
+        </ContentBox>
+      </GridItem>
+
+      <GridItem xl={12} xl2={6}>
+        <Toolbar>
+          <ToolbarItem>
+            <TextContent>
+              <Text component={TextVariants.h2}>Checksums</Text>
+            </TextContent>
+          </ToolbarItem>
+        </Toolbar>
+        <ContentBox borderTop padding contentBoxHeight={`calc(100% - 78px)`}>
+          <Attributes>
+            <AttributesItem title={artifactEntityAttributes.md5.title} forceStringWrap>
+              {serviceContainerArtifact.data?.md5}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.sha1.title} forceStringWrap>
+              {serviceContainerArtifact.data?.sha1}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.sha256.title} forceStringWrap>
+              {serviceContainerArtifact.data?.sha256}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.build.title}>
+              {serviceContainerArtifact.data?.build && (
+                <>
+                  <BuildName build={serviceContainerArtifact.data.build} long includeBuildLink includeConfigLink /> (
+                  <Link to={`/builds/${serviceContainerArtifact.data.build.id}`}>#{serviceContainerArtifact.data.build.id}</Link>)
+                </>
+              )}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.buildCategory.title}>
+              <Label color="grey">{serviceContainerArtifact.data?.buildCategory}</Label>
+            </AttributesItem>
```

Comment:
Build and Build Category are not parts of the Checksums section and were on the left side(Details section) in our old UI, is there any reason for moving it?

Reply:
Yes, we agreed to move it to keep it more consistent with ArtifactList component, but the main section title should be renamed to `Checksums and Build`